src/sql/src/statements/create.rs

```diff
@@ -310,6 +312,9 @@ impl Display for CreateView {
             write!(f, "IF NOT EXISTS ")?;
         }
         write!(f, "{} ", &self.name)?;
+        if !self.columns.is_empty() {
+            write!(f, "({}) ", format_list_comma!(self.columns))?;
+        }
```

Comment:
do we need some tests to reflect this change?

Reply:
Yes, let me test it in parser.